src/StoragelessSession/Session/Data.php

```diff
@@ -33,14 +33,50 @@ class Data implements \JsonSerializable
     private $metadata;
 
     /**
+     * @var bool
+     */
+    private $dataWasChanged = false;
```

Comment:
@lcobucci yeah! But for now, It's private, so that `externally` cannot access this. It's only class scope purpose.
What do you think? Still wrong?